VIDEO-6263: low fps from safari with simulcast enabled. (#1560)
twilio/twilio-video.js: CHANGELOG.md

**Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.

2.17.0 (In Progress)

> ====================

Reviewer: ```suggestion
====================

```

Bug Fixes

> ---------

Reviewer: ```suggestion
---------

```

> Fixed a bug that causes low framerate from Safari participants with simulcast enabled. (VIDEO-6263)

Reviewer: ```suggestion
- Fixed a bug where the VideoTracks of Safari Participants with VP8 simulcast enabled sometimes had low frame rates. (VIDEO-6263)
```
